ports/darksheep/port.json

```
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "darksheep.zip",
+  "items": [
+    "Dark Sheep.sh",
+    "darksheep"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Dark Sheep",
+    "porter": [
+      "Kompaan"
+    ],
+    "desc": "Dark Sheep is a block-pushing puzzle game with horror elements, and a love letter to the Commodore 64, both in visuals and audio! Plays like classic Sokoban with new twists that force you to find creative solutions!",
+    "desc_md": null,
+    "inst": "Grab the windows version from https://daisy-games.itch.io/dark-sheep\nand copy DarkSheep.pck into /roms/ports/darksheep/gamedata\n\nText is harder to read on 320p, but playable. Works best on 480p and up.",
```

muOS has the ports placed in the ports folder at the root which in turn mounts as /mnt/mmc/ports for the 1st memory stick or /mnt/sdcard/ports for the 2nd memory stick so /roms/ports (which could also be /roms2/ports for the 2nd memory stick on ArkOS, for example) isn't absolute.
```suggestion
    "inst": "Grab the windows version from https://daisy-games.itch.io/dark-sheep\nand copy DarkSheep.pck into ports/darksheep/gamedata\n\nText is harder to read on 320p, but playable. Works best on 480p and up.",
```